### DISCUSSION TITLE
Select world seed as env variable

### DISCUSSION THREAD

**Discussion by chlorophant** [mbround18/valheim-docker#173] [Ideas]
The readme does not clearly state which if any of the variables correlate to a world seed. I would like to use "Odinisking" seed. 

I set: WORLD="Odinisking"

However the server startup logs indicate it is not being used.

02/24/2021 18:04:37: Initializing world generator seed:YVfxTiKNFD ( 1611789227 )   menu:False  worldgen version:1

Is there a variable that isnt documented or what?

**Comment by CosmicHorrorDev** [comment]
`WORLD` isn't the world seed. AFAIK from the limited amount of information provided in the valheim dedicated server getting started guide there isn't a way of specifying a seed (I'd love to be corrected here. I looked into passing in the seed as an env var a bit before)

**Comment by mbround18** [comment]
[Valheim Dedicated Server Manual.pdf](https://github.com/mbround18/valheim-docker/files/6039632/Valheim.Dedicated.Server.Manual.pdf)

**Comment by Cuel** [comment]
You could generate the world in your client and transfer it to the server.
